terraform/application/config/production/variables.tfvars.json

```diff
@@ -7,5 +7,10 @@
   "enable_monitoring": true,
   "external_hostname": "apply-for-qts-in-england.education.gov.uk",
   "namespace": "tra-production",
-  "uploads_storage_environment_tag": "Production"
+  "uploads_storage_environment_tag": "Production",
+  "azure_maintenance_window": {
+    "day_of_week": 0,
+    "start_hour": 3,
+    "start_minute": 0
+  }
```

Comment:
Can we keep these variables in alphabetical order to make them easier to find?

Reply:
fixed